fetch/api/request/destination/fetch-destination-no-load-event.https.html

```diff
@@ -121,4 +121,11 @@
     await waitOnMessageFromSW(t);
 }, '@import fetches with a "style" Request.destination');
 
+// worker destination
+/////////////////////
+promise_test(async t => {
+    let worker = new frame.contentWindow.Worker("dummy?t=worker&dest=worker");
```

Comment:
Current implementations incorrectly inherit the service worker controller from the parent document for worker scripts.  The spec says they should do their own matching against the scope.  Since this is kind of besides the point of this test it would be nice if you could make this work for either case.

If you change the scope to `resources/dummy` without the trailing .html and search params then I think this test will be valid for both current implementations and what the spec says.  It will do a substring match for both the iframe and then worker script.

You would of course need to change the `iframe.src = kScope` to set the html suffix and search params as well.